src/native/corehost/hostpolicy/static/coreclr_resolver.cpp

```diff
@@ -8,6 +8,8 @@
 #include <pal.h>
 #include <trace.h>
 
+typedef void (*coreclr_error_writer_callback_fn) (const char *message);
```

Comment:
I don't think this is necessary - the same typedef already exists in `coreclr_resolver.h` which is included.

Reply:
Removed in the last commit.